source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -5,4 +5,107 @@ Access Data From a Cursor
 .. default-domain:: mongodb
 
 Read operations that return multiple documents do not immediately return
-all values matching the query.
+all values matching the query. Because a query can potentially match
+large sets of documents, these operations rely upon an object called a
+:java-docs:`MongoCursor </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCursor.html>`.
```

Comment:
nit: This produces a url with a double `//` `https://mongodb.github.io/mongo-java-driver/4.2//apidocs/mongodb-driver-sync/com/mongodb/client/MongoCursor.html` .

Change to: `` `:java-docs:`MongoCursor <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCursor.html>` ``